### PR TITLE
docs: Add readme

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,329 @@
...
+This is Appium driver for automating macOS applications using Apple's [XCTest](https://developer.apple.com/documentation/xctest) framework.
+The driver operates in scope of [W3C WebDriver protocol](https://www.w3.org/TR/webdriver/) with several custom extensions to cover operating-system specific scenarios.
+The original idea and parts of the source code are borrowed from the Facebook's [WebDriverAgent](https://github.com/facebookarchive/WebDriverAgent) project.
+
+
+## Requirements
+
+On top of standard Appium requirements Mac2 driver also expects the following prerequisites:
+
+- Mac OS 10.15 or later
```
Comment (KazuCocoa) [inline README.md:13]: Good to know. I thought this was since Big Sur... :P

File: README.md [hunk 2]
```diff
@@ -1,4 +1,329 @@
...
+Perform click gesture on an element or by relative/absolute coordinates
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+element | string | if `x` or `y` are unset | Unique identifier of the element to perform the click on. Either this property or/and x and y must be set. If both are set then x and y are considered as relative element coordinates. If only x and y are set then these are parsed as absolute coordinates. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
+x | number | if `y` is set or `element` is unset | click X coordinate | 100
+y | number | if `y` is set or `element` is unset | click Y coordinate | 100
+keyModifierFlags | number | no | if set then the given key modifiers will be applied while click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | 1 << 1 `logical or` 1 << 2
```
Comment (KazuCocoa) [inline README.md:84]: `1 << 1` logical or `1 << 2`? (for example of keyModifierFlags)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,329 @@
...
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+element | string | if `x` or `y` are unset | Unique identifier of the element to perform the long click on. Either this property or/and x and y must be set. If both are set then x and y are considered as relative element coordinates. If only x and y are set then these are parsed as absolute coordinates. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
+x | number | if `y` is set or `element` is unset | long click X coordinate | 100
+y | number | if `y` is set or `element` is unset | long click Y coordinate | 100
+duration | number | yes | The number of float seconds to hold the mouse button | 2.5
+keyModifierFlags | number | no | if set then the given key modifiers will be applied while long click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | 1 << 1 `logical or` 1 << 2
```
Comment (KazuCocoa) [inline README.md:98]: `1 << 1` logical or `1 << 2`? (for example of keyModifierFlags. The positions of `)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,329 @@
...
+
+Start an app with the given bundle identifier or activates it if the app is already running. An exception is thrown if the app with the given identifier cannot be found.
+This API influences the state of the [Application Under Test](#application-under-test-concept).
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+bundleId | string | yes | bundle identifier of the app to be launched or activated | com.apple.TextEdit
+arguments | array | no | the list of command line arguments for the app to be be launched with. This argument is ignored if the app is already running. | ['--help]
```
Comment (KazuCocoa) [inline README.md:224]: "['--help]" -> "['--help']"? (is ' missing after `help`?)

File: README.md (hunk 2 above)
Comment (mykola-mokhnach) [reply README.md:84]: there is no error there

File: README.md (hunk 4 above)
Comment (mykola-mokhnach) [reply README.md:224]: fixed

File: README.md [hunk 5]
```diff
@@ -81,7 +81,7 @@ Name | Type | Required | Description | Example
 element | string | if `x` or `y` are unset | Unique identifier of the element to perform the click on. Either this property or/and x and y must be set. If both are set then x and y are considered as relative element coordinates. If only x and y are set then these are parsed as absolute coordinates. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
 x | number | if `y` is set or `element` is unset | click X coordinate | 100
 y | number | if `y` is set or `element` is unset | click Y coordinate | 100
-keyModifierFlags | number | no | if set then the given key modifiers will be applied while click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | 1 << 1 `logical or` 1 << 2
+keyModifierFlags | number | no | if set then the given key modifiers will be applied while click is performed. See the official documentation on [XCUIKeyModifierFlags enumeration](https://developer.apple.com/documentation/xctest/xcuikeymodifierflags) for more details | `1 << 1 | 1 << 2`
```
Comment (KazuCocoa) [inline README.md:84]: ah, got it

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**